### DISCUSSION TITLE
Push notification for Releases

### DISCUSSION THREAD

**Discussion by ionut-maxim** [community/community#10311] [Mobile] [answered]
Hey, I think adding the option to receive push notifications when a new release for a repo you're already watching would be nice.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @ionut-maxim — I've opened a tracking issue for this, and we'll see how we can prioritize it in the future 😊

**Reply by ionut-maxim** [reply]
Thanks for the update

**Reply by Tinker42** [reply]
If possible, could an option be added in the Custom Notifications area of the Watch drop-down for Commits? The releases option is great, but it doesn't trigger notifications for a repo I watch that doesn't use the Publish Release feature.

**Reply by mateuszdrab** [reply]
To be fair a lot of notifications options are missing, I'd like to get notified on new releases, comments on issues and PRs I follow, maybe even commits on some projects. Just some notification basics I'd say
